web/packages/core/src/ruffle-player.ts

```diff
@@ -1952,13 +1952,22 @@ export class RufflePlayer extends HTMLElement {
             this.isExtension &&
             window.location.origin !== this.swfUrl!.origin
         ) {
+            const url = new URL(this.swfUrl!);
+            if (this.loadedConfig?.parameters) {
+                const parameters = sanitizeParameters(
+                    this.loadedConfig.parameters
+                );
+                Object.keys(parameters).forEach((key) => {
```

Comment:
nit: I think `Object.entries()` can be used here to iterate on both keys and values, then `parameters[key]!` turns into just `value`.

Reply:
Good point, I've made the changes.